### PR TITLE
Fix footer not sticky on the bottom

### LINKED ISSUE #1171: Footer does't stick to the bottom when we are on the new post page
## Context
Context: I am access the website using the Opera web browser from my M1 Macbook Air.
Issue: Whenever we open the "New Post", the Footer keeps floating next to the post textfield, instead of being on the bottom of the page.

image

## Expected Behavior

I don't know if this is the expected behavior, but as an user it get me wonder if it was supposed to be like that... since generally speaking the Footer keeps sticked to the bottom of the page.

### PR DESCRIPTION
**Pull request by lindotex**
`#` ✨ Codu Pull Request 💻

Fixes #1171 

## Pull Request details
- Added an min height to the post component. (tiny small change), because I didn't want to make any huge alterations on the code.

## Any Breaking changes

- None

## Associated Screenshots

Desktop:
desktop view of the page

Mobile:
mobile view of the page

### CODE REVIEW

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]
The env should not be deleted and the min-h-96 might be too high or low depending on the screen size.

**Comment by NiallJoeMaher** [comment]
Just need `npm run prettier:fix`

File: app/(app)/layout.tsx [hunk 1]
```diff
@@ -79,13 +79,15 @@ export default async function RootLayout({
 
   return (
     <>
```
Comment (NiallJoeMaher) [inline app/(app)/layout.tsx:81]: No longer need the fragments. Let me know if you have time to update this.

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]

**Comment by NiallJoeMaher** [comment]
You can reopen if you decide to tackle again.

**Comment by lindotex** [comment]
Sorry by my late answer.
I was Traveling 😅...
I have the code ready.

**Comment by Zaid-B24** [comment]
I can work on this issue, if no one has any problem with it.

**Pull request closed without merging**